lib/tasks/plugin.rake

```diff
@@ -104,7 +104,7 @@ task 'plugin:update', :plugin do |t, args|
   end
 
   update_status = system("git -C '#{plugin_path}' pull")
-  abort('Unable to pull latest version of plugin') unless update_status
+  abort("Unable to pull latest version of plugin #{plugin_path}") unless update_status
```

Comment:
There's a second `abort` message on line 98.

Reply:
Good catch, thank you!